<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillInfoWidget</class>
 <widget class="QWidget" name="SkillInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>54</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hLayoutSkillInfoWidget">
   <item>
    <layout class="QVBoxLayout" name="vLayoutSkill"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod1">
     <property name="title">
      <string>1트포</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod1"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod2">
     <property name="title">
      <string>2트포</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod2"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod3">
     <property name="title">
      <string>3트포</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
